File: Frontend-Deploy-Center/src/types/SystemConfig.ts
export interface SystemConfig {
    id: number
    config_name: string
    config_key: string
    config_value: string | number | boolean
    config_remark: string
    config_group: string
    created_at?: Date | null; // 创建时间
    updated_at?: Date | null; // 更新时间
  }

File: Backend-Deploy-Agent/src/models/entity/system_config.py
from pydantic import BaseModel
from typing import Optional, Union
from datetime import datetime

class SystemConfig(BaseModel):
    id: int
    config_name: str
    config_key: str
    config_value: Union[str, bool, int, float, None]  # 支持多种类型
    config_remark: str
    config_group: str
    created_at: Optional[datetime] = None
    updated_at: Optional[datetime] = None